backend/routes.ts

```diff
@@ -78,6 +79,8 @@ const setup = ({
     next()
   })
 
+  const contentfulCache = cacheFactory(30) // 30 second TTL
```

Comment:
We possibly should make this configurable?

Reply:
Maybe, though I'm not sure we would gain that much? I guess if it was an env we would be able to pick it with out a new release - but we'd still have to to a redeploy for it to be picked up?